torch/csrc/jit/python/init.cpp

```diff
@@ -759,7 +759,11 @@ void initJITBindings(PyObject* module) {
         auto res =
             PyObject_CallMethod(buffer_.ptr(), "readinto", "O", memview.get());
         if (res) {
-          int64_t i = PyInt_AsLong(res);
+#ifdef _MSC_VER
+          int64_t i = PyLong_AsLongLong(res);
+#else
+          int64_t i = PyLong_AsLong(res);
```

Comment:
So specifically, I'm suggesting eliminating this if-branch of the ifdef and just always use the MSC_VER version.

Reply:
Then it will lead to a static casting here for Unix. Is that okay?